File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Farinha/06.pl
#Programa que calcula o IMC (Índice de Massa Corporal)
print "Digite o valor do IMC (Indice de Massa Corporal):\n";
$imc = <STDIN>;
if ($imc < 20)
{
	print "Abaixo do peso\n";
}
elsif ($imc >= 20 && $imc <= 24)
{
	print "Peso ideal\n";
}
elsif ($imc >= 25 && $imc <= 29)
{
	print "Excesso de peso\n";
}
elsif ($imc >= 30 && $imc <= 39)
{
	print "Obesidade\n";
}
elsif ($imc > 39)
{
	print "Obesidade morbida\n";
}
else
{
	print "Valor invalido\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Farinha/06.rb
#Programa que calcula o IMC (Índice de Massa Corporal)
puts "Digite o valor do IMC (Indice de Massa Corporal):"
imc = gets
if imc.to_i < 20
  puts "Abaixo do peso"
elsif imc.to_i >= 20 && imc.to_i <= 24
  puts "Peso ideal"
elsif imc.to_i >= 25 && imc.to_i <= 29
  puts "Excesso de peso"
elsif imc.to_i >= 30 && imc.to_i <= 39
  puts "Obesidade"
elsif imc.to_i > 39
  puts "Obesidade morbida"
else
  puts "Valor invalido"
end